docs/Guide/01-Prerequisites/2.md

```diff
@@ -0,0 +1,148 @@
+---
```

Comment:
Remove this from this PR and track in a new issue. We can do a follow-up PR where we present discovery for various types of client. This is very JS focused and hence does not belong in waku.guide.